x-pack/test/api_integration/apis/lens/existing_fields.ts

```diff
@@ -14,7 +14,7 @@ const TEST_END_TIME = '2023-09-23T18:31:44.000';
 const COMMON_HEADERS = {
   'kbn-xsrf': 'some-xsrf-token',
 };
-const metaFields = ['_id', '_index', '_score', '_source', '_type'];
```

Comment:
FYI this is the main change, everything else are functionals failing because there's one field less to expect.

Reply:
I think you meant this https://github.com/elastic/kibana/blob/6e8b8077498a4dd3e603a0ef1f237ad398ebcbd6/src/plugins/data/server/ui_settings.ts

Reply:
YES , this is the old version not the change :)